<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiApplicationClass</class>
 <widget class="QMainWindow" name="QtGuiApplicationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiApplication</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="submit_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>提交答案</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="answer_lineedit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>题目类型设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>260</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>开始做题</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="question_amount_lineedit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>210</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>210</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>题目数量：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>我的答案</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>倒计时：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前题目：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="history_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>390</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>答题历史记录</string>
    </property>
   </widget>
   <widget class="QLabel" name="index_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>231</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>当前题目：?? / ??</string>
    </property>
   </widget>
   <widget class="QLabel" name="score_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>201</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>当前得分： ??</string>
    </property>
   </widget>
   <widget class="QLabel" name="countdown_lable">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next_question_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>不会做，下一题</string>
    </property>
   </widget>
   <widget class="QLabel" name="right_wrong_label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>310</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGuiApplication.qrc"/>
 </resources>
 <connections/>
</ui>
